packages/firestore/src/local/local_store.ts

```diff
@@ -168,6 +169,13 @@ export class LocalStore {
     q.canonicalId()
   );
 
+  /**
+   * The read time of the last entry processed by `getNewDocumentChanges()`.
```

Comment:
`getDocumentChanges`?

Oh! I see: this refers to `getNewDocumentChanges` in `LocalStore` and not the one in the `RemoteDocumentCache`. This is slightly confusing and I have a slight preference for keeping the names the same, though up to you.

Reply:
I see your confusion:) I tried to be to smart about it and reverted back to using getNewDocumentChanges() everywhere, which still makes sense even when a read time is passed (at least I see it that way now).